iOSDFULibrary/Classes/Utilities/Streams/DFUStreamZip.swift

```diff
@@ -144,7 +145,7 @@ internal class DFUStreamZip : DFUStream {
                 }
                 
                 let softdeviceType = FIRMWARE_TYPE_SOFTDEVICE
-                if type & softdeviceType == softdeviceType {
+                if type.rawValue & softdeviceType == softdeviceType {
```

Comment:
Instead of using raw value can't we set `FIRMWARE_TYPE_SOFTDEVICE` to the equivalent enum of DFUFirmwareType ?
this is not a breaker but it would be a tiny bit cleaner

Reply:
see above